compiler/optimizer/LoopReplicator.cpp

```diff
@@ -331,6 +338,8 @@ int32_t TR_LoopReplicator::replicateLoop(TR_RegionStructure *region,
    TR::TreeTop *lastTT = cBlock->getLastRealTreeTop();
    if (!lastTT->getNode()->getOpCode().isBranch())
       {
+      TR::DebugCounter::incStaticDebugCounter(comp(), TR::DebugCounter::debugCounterName(comp(), "LoopReplicator/NoBranchFoundInLoop/%s/(%s)/region_%d",
+         comp()->getHotnessName(comp()->getMethodHotness()), comp()->signature(), region->getNumber()));
```

Comment:
These are getting pretty repetitive. Could they be moved into a function to clean up these sites? e.g.

    countUnreplicated("NoBranchFoundInLoop", region);